VERSION.txt

```diff
@@ -1 +1 @@
-4.7.0
+4.8.0
```

Comment:
While it is a bug fix, it also changes how we communicate with GA.  In the off chance anyone was using and relying on it, they'd need to redo their analytics, so it felt "bigger" to me than a simple bug fix.